test/test_DFT.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import pytest
+import time
+
+def dft(x):
+    N = len(x)
+    X = np.zeros(N, dtype=complex)
+    for i in range(N):
+        for n in range(N):
+            X[i]=X[i]+ x[n] * np.exp(-2j*np.pi*i*n/N)
+    return X
+
+@pytest.mark.skip(reason="Deactivated test, no analytical data to compare. Only checks visual behaviour.")
```

Comment:
Try to think something which could be tested. Maybe compare with an analytical known result.